server/util/file.py

```diff
@@ -163,7 +164,22 @@ def get_file_upload_url(
         }
 
 
-def get_standard_file_upload_request_params(request: Request) -> Tuple[str, str, str]:
+def get_audit_folder_path(election_id: str) -> str:
+    return f"audits/{election_id}"
+
+
+def get_jurisdiction_folder_path(
+    election_id: str,
+    jurisdiction_id: str,
+) -> str:
+    return f"{get_audit_folder_path(election_id)}/jurisdictions/{jurisdiction_id}"
+
+
+def validate_and_get_standard_file_upload_request_params(
+    request: Request,
+    validate_storage_path: Callable[[str], None],
+    validate_file_type: Callable[[str], None],
```

Comment:
As a reader, it's a little complicated to grok the call sites of this function. It seems like they are mostly doing the same thing, so I wonder if this function could have the following signature:

```
def validate_and_get_standard_file_upload_request_params(
  request: Request,
  expected_file_directory_path: str,
  expected_file_name_prefix: str,
  expected_file_extension: str | List[str]
  expected_file_type: str | List[str]
)
```

Also, I'm realizing that file types and file extensions are in a predefined mapping, so maybe we could pull that out into a constant and just pass in one of them.